telugu/telangana-telugu/Telanganatelugu.md

```diff
@@ -11,4 +11,6 @@
    అస్తిత్వాన్ని చాటుకోవడమే. 
    ఇదంతా కవి అంతర్గత, బాహ్య 
    వలసాధిపత్యాలను వ్యతిరేకించే 
-   క్రమంలో చేసే కవితా గానమే.
\ No newline at end of file
+   క్రమంలో చేసే కవితా గానమే.
+
+Starting with a prayer. Please click [here](telugu/prayer-telugu/prayer-telugu.md).
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.